self-api/src/main/java/com/selfxdsd/api/Admin.java

```diff
@@ -0,0 +1,16 @@
+package com.selfxdsd.api;
+
+/**
+ * Admin user in Self XDSD.
+ * @todo: Implement and test Admin user class.
+ */
+
```

Comment:
@nmonokov Please remove this empty new line.

Also, add the ``@since 0.0.61`` tag. If you wish, you can also add the ``@author`` tag with your name and e-mail.

Read this page to learn how to write a ``@todo``: https://docs.self-xdsd.com/microtasking.html
The ticket number is always the originating ticket, the one which you are currently solving. The estimation is usually 60min (microtasking).

In this case, the todo should look like:

``@todo #946:60min we need to do this and that``